Formula/publish.rb

```diff
@@ -13,7 +13,8 @@ class Publish < Formula
   end
 
   # https://github.com/JohnSundell/Publish#system-requirements
-  depends_on xcode: ["11.4", :build]
+  depends_on xcode: ["12.4", :build]
+  depends_on macos: :catalina
```

Comment:
```suggestion
  depends_on xcode: ["12.5", :build]
```
This needs Swift 5.4, which comes with Xcode 12.5.

Reply:
Do we have a workaround for Catalina build yet?

Reply:
Nope, no workaround for a Catalina build. Upstream also explicitly requires Big Sur, but we might be able to do a Catalina build with Swift 5.4.